### PR TITLE
Allow 'code' to be built if Node.JS has not already been installed

### PR DESCRIPTION
**Pull request by JL2210**
And remove unused `@version` variable.

### CODE REVIEW

File: packages/code.rb [hunk 1]
```diff
@@ -35,13 +33,15 @@ class Code < Package
...
+    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2 | grep -v none`
     system "nodebrew install 8.15.0 || true"
     system "nodebrew use 8.15.0"
     system "npm install gulp"
     system "yarn install"
     system "yarn run gulp vscode-linux-#{@arch}"
-    system "nodebrew use #{node_old}"
+    if node_old != ''
+      system "nodebrew use #{node_old}"
+    end
```
Comment (uberhacker) [inline packages/code.rb]: This is not going to work.  node_old will always be not equal to '' since you didn't chomp.

Comment (uberhacker) [reply packages/code.rb]: Have you even tried building binaries?  Apparently, not since `nodebrew use none` always errors out.

Comment (JL2210) [reply packages/code.rb]: So all I need to do is add `.chomp` at the end of the line that assigns `#{node_old}`, right?

Comment (uberhacker) [reply packages/code.rb]: Anytime you use commands with backticks, you need to chomp the end to get rid of the non-printable characters.

Comment (JL2210) [reply packages/code.rb]: Got it. So is that ALL I need to do?

Comment (uberhacker) [reply packages/code.rb]: Test it without any node versions installed and see what happens.

Comment (JL2210) [reply packages/code.rb]: Done.

**Comment by uberhacker** [comment]
The code package is completely broken on the stable channel.  I think you need more than this to fix it.

**Comment by JL2210** [comment]
@uberhacker True, but this fixes a mistake that I admittedly made.

File: packages/code.rb [hunk 2]
```diff
@@ -35,13 +33,15 @@ class Code < Package
...
   def self.build
-    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2`
+    node_old = `nodebrew ls | fgrep 'current: ' | cut -d' ' -f2 | grep -v none`.chomp
     system "nodebrew install 8.15.0 || true"
     system "nodebrew use 8.15.0"
     system "npm install gulp"
     system "yarn install"
     system "yarn run gulp vscode-linux-#{@arch}"
-    system "nodebrew use #{node_old}"
+    if node_old != ''
```
Comment (uberhacker) [inline packages/code.rb]: You don't need the ` | grep -v none`.  You could:
`system "nodebrew use #{node_old}" unless "#{node_old}" == "none"`
Also, another consideration is to remove this node version when complete, therefore not impeding on the users node space:
`system "nodebrew uninstall 8.15.0" unless "#{node_old}" == "v8.15.0"`

Comment (uberhacker) [reply packages/code.rb]: You might want to use a variable for "v8.15.0" also.

Comment (JL2210) [reply packages/code.rb]: Good idea. That sounds perfect.

Comment (JL2210) [reply packages/code.rb]: One step ahead of you there, @uberhacker.

Comment (uberhacker) [reply packages/code.rb]: How about?
```
    system "nodebrew install #{node_ver}" unless "#{node_ver_installed}" == "#{node_ver}"
    system "nodebrew use #{node_ver}" unless "#{node_old}" == "#{node_ver}"
```

Comment (JL2210) [reply packages/code.rb]: That works too.

**Pull request merged**